app/assets/javascripts/miq_novnc.js

```diff
@@ -1,6 +1,8 @@
 "use strict";
 var rfb;
 
+window.WEB_SOCKET_SWF_LOCATION = '/assets/noVNC/web-socket-js/WebSocketMain.swf';
```

Comment:
@chessbyte : we are the end user of the file, not a library that may need to allow applications to place their own override.

A side note: this is old browser fix that will not be needed once we drop IE9. I want it here to match downstream.